app/controllers/like_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikeController < ApplicationController
+  def new
+    @like = Like.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: @like } }
+    end
+  end
```

Comment:
[REQUIRED] Kindly remove the new action since you won't be needing a form just to add likes